### PR TITLE
Initial configuration for COUNTRY=bhutan

### PR DESCRIPTION
**Pull request by wadhwamatic**
### Description

This fixes #issueNumber.

## How to test the feature:

- [ ]
- [ ]

## Checklist - did you ...

Test your changes with

- [ ] `REACT_APP_COUNTRY=rbd yarn start`
- [ ] `REACT_APP_COUNTRY=cambodia yarn start`
- [ ] `REACT_APP_COUNTRY=mozambique yarn start`
- [ ] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

File: frontend/src/config/bhutan/layers.json [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+        "fill-opacity": 0
+      },
+      "line": {
+        "line-color": "gray",
+        "line-width": 1,
+        "line-opacity": 0.8
+      }
+    }
+  },
+  "rainfall_dekad": {
```
Comment (ericboucher) [inline frontend/src/config/bhutan/layers.json:40]: Let's rely on the shared config as much as possible here.
If we are hindered by admin areas/boundaries, let's make a note of it in https://github.com/WFP-VAM/prism-app/issues/1212

File: frontend/src/config/bhutan/prism.json [hunk 2]
```diff
@@ -0,0 +1,188 @@
+{
+  "country": "Bhutan",
+  "defaultLanguage": "en",
+  "countryAdmin0Id": 31,
+  "alertFormActive": false,
+  "hidePanel": false,
+  "enableNavigationDropdown": true,
+  "icons": {
```
Comment (ericboucher) [inline frontend/src/config/bhutan/prism.json:8]: Same, let's rely on the shared config as much as possible for things like that. Don't hesitate to add to the shared/prism.json

**Review by ericboucher — APPROVED** [review]

File: frontend/src/config/bhutan/layers.json (hunk 1 above)
Comment (wadhwamatic) [reply frontend/src/config/bhutan/layers.json:40]: @ericboucher - I added a comment in 1212 as suggested. I could add `chart_data` to `shared/layers.json` but the levels fields will be wrong. Should we add it anyway, or just skip that element in the shared config, knowing either will break the chart feature unless the local config addresses it?

Comment (ericboucher) [reply frontend/src/config/bhutan/layers.json:40]: let's keep it as you did for now I think, it's clearer

**Pull request merged**